.gitignore

```diff
@@ -58,3 +58,4 @@ docs/_build/
 
 # IDE
 .idea
+.vscode
```

Comment:
You should probably put this in your `~/.gitignore` instead, but at least leave it newline terminated.
(Yes, I have no idea how `.idea` got in there.  No, this isn't very important)